release/api/v1alpha1/bundle_test.go

```diff
@@ -0,0 +1,30 @@
+package v1alpha1_test
+
+import (
+	"testing"
+
+	"github.com/aws/eks-anywhere/release/api/v1alpha1"
+)
+
+func TestParseVersionFromBundleVersion(t *testing.T) {
+	tests := map[string]struct {
+		version string
+		want    string
+	}{
+		"DevBuildBundleVersion": {
+			version: "v0.0.0-dev+build.8033+04134fa",
+			want:    "v0.0.0-dev+build.8033",
+		},
+		"ProdBundleVersion": {
+			version: "v0.18.3+cc70180",
```

Comment:
Q: Have you figured out what this hash means?

Reply:
Yes, spoke to @abhay-krishna it's a hash of all the manifest contents in that bundle and we probably don't need it in the annotation.